docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.7"
+version: "3"
```

Comment:
3.7 version requires docker engine 18.06.0 and above.

This has networking features that are used in our cross repo services. It's a better practice to update to 3.7, it's almost 18 months since it is released.

One option is to update the documentation to make docker enginer a prerequisite.

Other one is the above fix, we make the version check easier. 

What do you think Sanket?

Reply:
I see.

> When specifying the Compose file version to use, make sure to specify both the major and minor numbers. If no minor version is given, 0 is used by default and not the latest minor version. As a result, features added in later versions will not be supported.

I had somehow completely missed this. Apologies, adding info to upgrade Docker Engine in the README seems the best way going forward.

Do you want me to go ahead and do it?

Reply:
Yes, please add the same in the README.